packages/block-editor/src/components/inserter/quick-inserter.js

```diff
@@ -82,15 +83,18 @@ export default function QuickInserter( {
 		}
 	}, [ setInserterIsOpened ] );
 
+	const { setInserterInsertionPoint } = unlock(
+		useDispatch( blockEditorStore )
+	);
+
 	// When clicking Browse All select the appropriate block so as
 	// the insertion point can work as expected.
 	const onBrowseAll = () => {
 		setInserterIsOpened( {
-			rootClientId,
-			insertionIndex,
 			filterValue,
 			onSelect,
 		} );
+		setInserterInsertionPoint( { rootClientId, insertionIndex } );
```

Comment:
We could have the editor stores `setInserterIsOpened` dispatch the `setInserterInsertionPoint` to the block editor store as part of the action. That said it might be clearer this way as the "set inserter is opened" communicates "toggling" rather than "setting an insertion point".

Reply:
But what about backwards compat on the `setInserterIsOpened` API?

Reply:
What I mean is that I think its good to have the flexibility of calling these independently rather than tying one to the other.